packages/components/src/components/input-field/input-field.tsx

```diff
@@ -122,7 +122,7 @@ const InputField = ({
 }: TInputField) => {
     const [local_value, setLocalValue] = React.useState<string>();
     const Icon = icon as React.ElementType;
-    const has_error = error_messages && !!error_messages.length;
+    const has_error = error_messages && !!error_messages.length && !is_error_tooltip_hidden;
```

Comment:
`is_error_tooltip_hidden` is passed from `<InputWithCheckbox />` as `is_error_tooltip_hidden={isMobile()}`